src/app/modules/shared/components/details-fields/details-fields.component.html

```diff
@@ -90,6 +90,7 @@
             aria-describedby="inputGroup-sizing-sm"
             [class.is-invalid]="start_date.invalid && start_date.touched"
             required
+            (ngModelChange)="onInitialDatepickerUpdated($event)"
```

Comment:
I think we can use a another name for this function `onInitialDatepickerUpdated` something like `onStartDateChange` or something similar.